### PR TITLE
 boac-5470 - fix add department in passenger manifest

### PR DESCRIPTION
**Pull request by mohamalnadi**
Jiras:

- https://jira-secure.berkeley.edu/browse/BOAC-5470

### CODE REVIEW

File: src/components/note/EditBatchNoteModal.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
     min-width="500"
     persistent
     width="60%"
-    @keyup.esc.prevent="discardRequested"
+    @keyup.esc.prevent
```
Comment (lyttam) [inline src/components/note/EditBatchNoteModal.vue]: @mohamalnadi I think you can remove this empty event handler.

Comment (mohamalnadi) [reply src/components/note/EditBatchNoteModal.vue]: Removed it, thank you!

**Comment by pauline2k** [comment]
Code looks good, @mohamalnadi! Would it be possible to split the fixes for these two different issues into two different PRs?

**Comment by mohamalnadi** [comment]
> Code looks good, @mohamalnadi! Would it be possible to split the fixes for these two different issues into two different PRs?

Thank you! yes! I made **this** PR for passenger manifest only and I just opened another PR for the notes fix: https://github.com/ets-berkeley-edu/boac/pull/4240

**Pull request merged**